rails/app/controllers/home_controller.rb

```diff
@@ -11,4 +11,8 @@ def show
   def create
     # save the story to the database
   end
+
+  helper_method def stories
+    [Story.new(title: 'Story Title', desc: 'Description', speaker: 'Speaker')] * 4
+  end
```

Comment:
Nice! From the screenshot I assumed the example stories were hard-coded in the view, but this is a much better stepping-stone to actually querying the model!